script.js

```diff
@@ -1,4 +1,137 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+// Use Math to generate system's generated input for the game
+const getRandomHand = () => {
+  const systemGenerateNumber = 1 + Math.floor(Math.random() * 3);
+  if (systemGenerateNumber === 1) {
+    return "scissors";
+  } else if (systemGenerateNumber === 2) {
+    return "paper";
+  } else return "rock";
+};
+
+// Convert user's text input into symbol for output display
+const iconForNoun = (hand) => {
+  if (hand.toLowerCase() === "paper") {
+    return "🧻";
+  } else if (hand.toLowerCase() === "scissors") {
+    return "✂️";
+  } else if (hand.toLowerCase() === "rock") {
+    return "💎";
+  } else return hand.toLowerCase();
```

Comment:
Good job for firstly picking up potential capital letters, and next, good on you to have found a way to decompose all your icons to tie together with you messages! Well done!!